### DISCUSSION TITLE
Not working for me

### DISCUSSION THREAD

**Discussion by Fovruf** [Kvan7/Exiled-Exchange-2#105] [General]
Hi, somehow the exiled exchange 2 is not (or only partially) working for me. The program itself runs, and changing things in the browser works just fine, however using any of the hotkey combinations simply don't do anything for me. In the debug section it does track my key presses, so I know it sees what I do, but it just doesn't act at all in the game. Changing the hotkeys to other keys for example doesn't seem to work. Any suggestions?

Small edit: Both standalone and the installed versions won't work ingame

**Comment by Kvan7** [comment]
Not quite sure, I would need more information on what your setup is. Like fullscreen mode, os, and if you had this issue in PoE 1

**Comment by Fluffums** [comment]
same problem, Fullscreen-windowed mode. OS Win10 DX12 tried on vulcan and DX11, bahaves the same way as mentioned above, ir showed some signs of llife for first item check and then did nothing

**Comment by Kvan7** [comment]
See if [this page](https://kvan7.github.io/Exiled-Exchange-2/nothing-happens.html) helps.
If you are seeing `warn [ClipboardPoller] No item text found.` then it can probably be fixed by stuff on that page

**Comment by Fovruf** [comment]
My setup is windows full screen, w10. Tried installed and portable, both not working. The plugin works like a charm in poe1, just refuses to open up in poe2.

I checked the page you linked. Ctrl-C works fine. I pasted the text in notepad and it shows the full item name, stats etc. And like I said earlier I see the program noticing my key presses when I enable the debug section, it just doesn't do anything somehow.

**Reply by Kvan7** [reply]
> windows full screen

I'm assuming you meant windowed full screen? If not see [requirements](https://kvan7.github.io/Exiled-Exchange-2/download), exclusive full screen is not supported

> just refuses to open up in poe2

Did you copy your settings from PoE1? If so make sure you changed the window title in the config.json to be "Path of Exile 2"

**Reply by Fovruf** [reply]
Yeah windowed full screen, wrote it wrong my bad.

And I did copy over the poe1 settings, and apparently missed that part. That did the trick for me thank you!

**Reply by Kvan7** [reply]
Awesome! Glad that worked!
